docs/overview.yaml

```diff
@@ -414,11 +414,35 @@ Geometry:
   Differentiable manifolds:
     smooth or analytic manifold (with boundary and corners): 'IsManifold'
     smooth map between manifolds: 'ContMDiff'
+    product of manifolds: 'ModelWithCorners.prod'
+    interior of a manifold: 'ModelWithCorners.interior'
+    boundary of a manifold: 'ModelWithCorners.boundary'
     tangent bundle: 'TangentBundle'
     tangent map: 'tangentMap'
+    pullback of vector fields: VectorField.pullback
+    Lie bracket of vector field: VectorField.mlieBracket
+    Jacobi/Leibniz identity: VectorField.leibniz_identity_mlieBracket
+    integral curves: 'IsIntegralCurve'
+    local existence of integral curves: exists_isIntegralCurveAt_of_contMDiffAt_boundaryless
+    uniform time existence: exists_isIntegralCurve_of_isIntegralCurveOn
+    # corollary for compact manifolds PRed, but not yet merged
+    uniqueness of integral curves: 'isIntegralCurveOn_Ioo_eqOn_of_contMDiff_boundaryless'
     Lie group: 'LieGroup'
     sphere: 'EuclideanSpace.instIsManifoldSphere'
 
+  Smooth vector and fiber bundles:
+    (topological) fiber bundle: FiberBundle
+    (topological) vector bundle: VectorBundle
+    (topological) trivial vector bundle: Bundle.Trivial.vectorBundle
+    (topological) direct sum of vector bundles: VectorBundle.prod
+    (topological) pullback vector bundle: VectorBundle.pullback
+    (topological) Hom bundle: Bundle.ContinuousLinearMap.vectorBundle
+    smooth vector bundle: ContMDiffVectorBundle
+    (smooth) trivial vector bundle: Bundle.Trivial.contMDiffVectorBundle
+    (smooth) direct sum of vector bundles: Bundle.Prod.contMDiffVectorBundle
+    (smooth) pullback vector bundle: ContMDiffVectorBundle.pullback
+    (smooth) Hom bundle: ContMDiffVectorBundle.continuousLinearMap
+
```

Comment:
I have no idea about differential geometry specifically, but if you're looking for ways of condensing this, then once #23801 lands you could do something like:

```
    Hom bundle:
      topological: Bundle.ContinuousLinearMap.vectorBundle
      smooth: ContMDiffVectorBundle.continuousLinearMap
```

Reply:
Good idea, thanks! I've used that idea now :-)